code/__DEFINES/factions.dm

```diff
@@ -19,6 +19,7 @@
 #define FACTION_PIRATE "Pirate"
 #define FACTION_VALHALLA "Valhalla"
 #define FACTION_SPECFORCE "Special Forces"
+#define FACTION_CRASH "Crash"
```

Comment:
I know it works, but this seems like a real misuse of faction lol

Reply:
agree lol

Reply:
It's the cleanest way to get it done + `FACTION_VALHALLA` exists. I could seperate the vendor/loadout defines and these faction defines, though.